### ISSUE TITLE
[GHC] Workshop 1 Feeback - Your Name

### ISSUE THREAD

**Issue by myexampl35-dotcom** [finos/a11y-theme-builder#1211]
# Session 1

# Problems
1. [Theme Builder doesn’t reference the WCAG rules required for each style & component](https://github.com/finos/a11y-theme-builder/issues/1037)

2. [Transition between Theme Builder and Figma and code is complex](https://github.com/finos/a11y-theme-builder/issues/1038)

3. [Theme Builder does not offer a clear solution for users with existing coded components](https://github.com/finos/a11y-theme-builder/issues/1039)

*Select one problem above to work on this session. Delete the two problems that you won't focus on*

## How Might We

The facilitator will prompt you to convert the problem that was chosen to opportunities using the How Might We (HMW) method

1. How might we...
2. How might we...
3. How might we...
4. How might we...
5. How might we...
6. How might we...
7. How might we...
8. How might we...
9. How might we...
10. How might we...

After you have at least 10 HMWs, select the one that you believe has the most potential to generate impact

Make bold the HMW that you selected. 

## Brainstorm

Bring your HMW and post it here: ""

Brainstorm on possible solutions for the selected HMW and post one solution per numbered list item.

### Scratch pad
All ideas are good ideas!

1. Solution 1:
2. Solution 2:
3. Solution 3:
4. Solution 4:
5. Solution 5:
6. Solution 6:
7. Solution 7:
8. Solution 8:
9. Solution 9:
10. Solution 10:

Before you go, make bold your top three solutions.

CONGRATS! You have become an Accessibility Theme Builder contributor.
---

Join our Slack: https://github.com/finos/a11y-theme-builder/wiki/Communication#communication-channels

Join our weekly calls: https://github.com/finos/a11y-theme-builder/wiki/Communication#meetings

Contribute: Reach out to a maintainer and get started today!
